README.md

```diff
@@ -60,7 +60,7 @@ placed; if not specified the source directory is used for output as well.
 
 For all options, specify `oryx --help`.
 
-### `oryx -appPath`
+### `oryx script -appPath`
```

Comment:
nit: we can disagree, but have you thought about createscript or something like that instead of script??